Makefile

```diff
@@ -160,3 +160,6 @@ pathfinder: juno-cached
     	--p2p-peers=/ip4/127.0.0.1/tcp/8888/p2p/12D3KooWF1JrZWQoBiBSjsFSuLbDiDvqcmJQRLaFQLmpVkHA9duk \
     	--p2p-private-key="54a695e2a5d5717d5ba8730efcafe6f17251a1955733cffc55a4085fbf7f5d2c1b4009314092069ef7ca9b364ce3eb3072531c64dfb2799c6bad76720a5bdff0" \
     	--metrics-port=9094
+
+test-fuzz: ## run fuzzing script
```

Comment:
should we start running those in CI as well?

Reply:
We can, but there are some caveats:
- It runs one test at a time, which means the duration is $$n\times k$$ where $$n$$ - seconds for one test, $$k$$ - number of tests.
- If it fails, it creates a file with fail input and this test will be executed in case of `go test`.

Reply:
@derrix060 there is something called "OSS-Fuzz". As far as I understand it's 24/7 fuzzing infra - you can use something like this I guess. See https://github.com/golang/go/issues/50192 for more info